samza-yarn/src/test/scala/org/apache/samza/job/yarn/TestSamzaYarnAppMasterService.scala

```diff
@@ -75,8 +77,9 @@ class TestSamzaYarnAppMasterService {
     val jobModelManager = JobModelManager(config)
     val samzaState = new SamzaApplicationState(jobModelManager)
     val state = new YarnAppState(-1, ConverterUtils.toContainerId("container_1350670447861_0003_01_000002"), "testHost", 1, 1);
+    val registry = new MetricsRegistryMap()
```

Comment:
Oh cool. Thanks for fixing it :) It seemed like a one-off change. that's why I asked.